packages/autocomplete-core/src/propGetters.ts

```diff
@@ -228,7 +226,7 @@ export function getPropGetters<TItem>({
           providedProps.inputElement ===
             props.environment.document.activeElement &&
           !store.getState().isOpen &&
-          store.getState().query.length >= props.minLength
+          props.openOnFocus
```

Comment:
This kind of sounds funny to me.

* when it's not open
* and it's mean to be open on focus
* then let's give it a focus
* _(so that it will be open)_

I don't have any other idea. So we can leave it as is.

Reply:
I think this needs to be: `(props.openOnFocus || store.getState().query.length > 0)`.